innersource-portal.md

```diff
@@ -26,11 +25,14 @@ You are attempting to establish an InnerSource practice within your organization
 
 ## Context
 
+#### Prerequisites
+
 * Managers have given tacit acceptance that their employees should take part in InnerSource projects
-* InnerSource project owners are seeking a way to increase participation in their projects
 * A shared source control management system is in use which provides programmatic access to the contents of the repositories it hosts
-* The source control management application in use has limited search capabilities
-* There is a department within the organization with the responsibility of promoting InnerSource collaboration 
+* There is a department within your organization with the responsibility of promoting InnerSource collaboration
+
+Your organization is interested in adopting an InnerSource work style and InnerSource project owners are seeking a way to attract audiences to their projects. However, they are limited by the communication channels available to them through which they could advertise to potential contributors.  Compounding this problem is the fact that the shared source control management application in use has such limited search capabilities that even developers in search of InnerSource projects find it frustrating to locate them.
```

Comment:
I think that this paragraph should go above **Prerequisites** otherwise it looks like this content is part of the prerequisites.  It's actually the part of the context that is going to change (right)?